### ISSUE TITLE
[Bug]: KServe microcopy

### ISSUE THREAD

**Issue by lucferbux** [opendatahub-io/odh-dashboard#2178]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Deploy type

OpenDataHub core version (eg. `v1.6.0`)

### Version

2.5

#### Replace all "Single model" references with "Single-model" for KServe

Right now, all the references for KServe are displayed as "Single model", but all the ModelMesh references are marked as "Multi-model" with a hyphen. 

<img width="416" alt="Screenshot 2023-11-16 at 11 04 34" src="https://github.com/opendatahub-io/odh-dashboard/assets/16117276/4173d67a-1dd8-44fc-864e-489bb7cec5c9">

All the references for KServe to be named "Single-model" with a hyphen.

1. Go to the Custom Serving Runtimes section
2. If you have kserve enable, the tag should display "Single model"
3. If you create a new model, you'll get "Single model"

### Expected Behavior

Expected behavior in the description

**Comment by bdattoma** [comment]
@lucferbux what about changing the text which appear when user disables model serving platforms?

![image](https://github.com/opendatahub-io/odh-dashboard/assets/88311595/15910d1e-09a4-48f3-8ea0-41db16a36f3a)
![image](https://github.com/opendatahub-io/odh-dashboard/assets/88311595/8ba70843-2715-4d44-a7aa-d7000eff33d7)


I feel it's difficult to understand the different btw the 2 modes from these messages:
- too many words
- the two messages are too similar
- the message explains the opposite mode instead of the one which just got disabled - I can see the logic behind the choice, can make sense. But in the complex I find it a bit confusing
-  the fact that the messages appear only after disabling the modes doesn't help IMO

Moreover I can see users saying "oh, I want to deploy multiple models, not only one! so I select multi-model" - I think a such choice should be helped in UI rather than requiring to read docs, which are good for getting more details if they want


I would suggest the following changes:
1. describe the 2 options, maybe under a  tooltip (`?` icon)
2. reduce the text
3. when users disable one of the option, a short message appears informing that the new setting is going to affect only new projects

Wdyt?

**Comment by Xaenalt** [comment]
Is there a compelling reason to hide the technology names? My vote would be to call them KServe and ModelMesh tbh

**Comment by vconzola** [comment]
@Xaenalt Hiding the technology names was part of the requirements for KServe from the BU.

**Comment by Xaenalt** [comment]
Ok, bit odd, but yeah on reflection on the copy overall, it does seem a bit confusing to the user, I think most users would probably read the titles and select multi-model serving just based on the info there, and since KServe is the overall future direction, we may want to update the language to prefer KServe somehow

**Comment by vconzola** [comment]
@Xaenalt For new installs KServe=ON, MM=OFF is the default selection. That should show preference for KServe. Also, this is an admin UI. Presumably, anyone working in it should have some idea of what they're doing. That said, we can improve the microcopy of the alerts to improve their readability. We can also add some additional text below each radio button option to better explain what the difference is. We'll likely add something similar to what we show on the platform selection cards when both single and multi serving are enabled (see below).

![image](https://github.com/opendatahub-io/odh-dashboard/assets/22823035/92e885cf-6a8c-487f-9a97-0a8df22895f0)

**Comment by Xaenalt** [comment]
Maybe even Advanced Model Serving or Basic Model Serving or something, given the confusion with single/multi language, idk

**Comment by israel-hdez** [comment]
I can see that the "multi-model" and "single-model" wording can be confusing. I do think the most important point that can lead to mistakes is the one that @bdattoma mentions:

> Moreover I can see users saying "oh, I want to deploy multiple models, not only one! so I select multi-model" - I think a such choice should be helped in UI rather than requiring to read docs, which are good for getting more details if they want

Like @Xaenalt pointed, I'm not sure if it is already too late to change terminology. Given the descriptions I can read in the shared images (specifically, [this one](https://user-images.githubusercontent.com/16117276/283433192-b93645f6-dac4-433c-8c8c-f9e59384335d.png)), if there is still a chance to change terminology, I'd suggest to change:
* Multi-model serving platform -> Shared model server orchestrator
* Single-model serving platform -> Dedicated model server orchestrator

...or something like that... trying to leave out the "platform" word, assuming RHODS is the platform.

IMHO, the shared/dedicated terminology will reflect with more accuracy what we are trying to communicate in the UI.

**Comment by andrewballantyne** [comment]
@kaedward is out this week. So this may need to delay for the next release of KServe instead of the first one we are running at. 

cc @dgutride @lucferbux

**Comment by kaedward** [comment]
@vconzola @andrewballantyne @lucferbux 

Lots of potential changes! I'll try to address everything mentioned here, but LMK if I miss anything or have gotten anything mixed up... 
In these suggestions, I’m also taking into account the [conversation](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1701185416032089) that took place on slack. This is a lot of info, and I'm finding it difficult to cut length without losing key points that users might need.

**1. Model serving platform names** 
@israel-hdez If this change is more clear and just as accurate, I don't see a problem with it. But we will need to update this terminology everywhere. I'm also not sure if it's too late to change this, though. 
Multi-model serving platform -> Shared model server orchestrator
Single-model serving platform -> Dedicated model server orchestrator

**2. Model serving platform descriptions**
Single model serving platform:
Each model is deployed from its own model server. Only use this option for large language models that will be deployed using the Caikit runtime.

Multi-model: 
Multiple models are deployed from the same model server. Use this option when deploying small models that can share server resources.

**3. Disabling model serving platforms info alerts**
Single is disabled: 
Disabling dedicated model serving prevents large language models from being deployed from projects on this cluster. You will only be able to deploy smaller models that can share a model server. Existing projects with deployed models will continue using their selected serving orchestrator.

Multi is disabled: Disabling shared model serving prevents models from being deployed from shared model servers. Instead, they will use dedicated model servers. Existing projects with deployed models will continue using their selected serving orchestrator.

**Comment by Xaenalt** [comment]
Yeah, upon reading it, I like the shared/dedicated model server orchestrator terminology

**Comment by vconzola** [comment]
I'm ok with "shared" and "dedicated" but the term "platform" (vs. orchestrator) comes directly from the [KServe Github page](https://github.com/kserve/kserve) - "KServe is a standard, cloud agnostic Model Inference Platform on Kubernetes, built for highly scalable use cases."

**Comment by israel-hdez** [comment]
> the term "platform" (vs. orchestrator) comes directly from the [KServe](https://github.com/kserve/kserve)

Well... we already have platforms inside platforms :wink: 
Use what fits best for the cards :-)

**Comment by kaedward** [comment]
@vconzola thanks! Let's keep "platform" then :-)
Updated suggestions:

1. Model serving platform names
Multi-model serving platform -> Shared model server platform
Single-model serving platform -> Dedicated model server platform

2. Model serving platform descriptions
Single model serving platform:
Each model is deployed from its own model server. Only use this option for large language models that will be deployed using the Caikit runtime.

Multi-model:
Multiple models are deployed from the same model server. Use this option when deploying small models that can share server resources.

3. Disabling model serving platforms info alerts
Single is disabled:
Disabling dedicated model serving prevents large language models from being deployed from projects on this cluster. You will only be able to deploy smaller models that can share a model server. Existing projects with deployed models will continue using their selected serving platform.

Multi is disabled: Disabling shared model serving prevents models from being deployed from shared model servers. Instead, they will use dedicated model servers. Existing projects with deployed models will continue using their selected serving platform.

**Comment by bdattoma** [comment]
> @vconzola thanks! Let's keep "platform" then :-) Updated suggestions:
> 
> 1. Model serving platform names
>    Multi-model serving platform -> Shared model server platform
>    Single-model serving platform -> Dedicated model server platform
> 2. Model serving platform descriptions
>    Single model serving platform:
>    Each model is deployed from its own model server. Only use this option for large language models that will be deployed using the Caikit runtime.
> 
> Multi-model: Multiple models are deployed from the same model server. Use this option when deploying small models that can share server resources.
> 
> 3. Disabling model serving platforms info alerts
>    Single is disabled:
>    Disabling dedicated model serving prevents large language models from being deployed from projects on this cluster. You will only be able to deploy smaller models that can share a model server. Existing projects with deployed models will continue using their selected serving platform.
> 
> Multi is disabled: Disabling shared model serving prevents models from being deployed from shared model servers. Instead, they will use dedicated model servers. Existing projects with deployed models will continue using their selected serving platform.


Thanks for the suggestions, I like them. I have only some thoughts about them:

1. `Only use this option for large language models that will be deployed using the Caikit runtime.` this is likely going to change when we'll support multiple runtimes. So I would not mention Caikit runtime specifically
2. `Only use this option for large language models ...` why? Customers have the freedom to import their own Custom Serving Runtime and try deploying other non-LLM models. I'd rather re-write it into a "suggested usage" of the platform.
3. I feel like the alerts on disabling are too long and, especially, implicit if we have already provided a description of each platform. I think it would be cleaner just to say  `Existing projects with deployed models will continue using their selected serving platform`

Wdyt?

**Comment by dgutride** [comment]
Migrated to https://issues.redhat.com/browse/RHOAIENG-543
